macros/unpack/get_metrics.sql

```diff
@@ -30,7 +34,7 @@
             "cast(" ~ dbt_project_evaluator.is_not_empty_string(node.description) | trim ~ " as boolean)",
             wrap_string_with_quotes(node.type),
             wrap_string_with_quotes(node.model.identifier),
-            wrap_string_with_quotes(node.label),
+            wrap_string_with_quotes(label),
```

Comment:
```suggestion
            wrap_string_with_quotes(dbt.escape_single_quotes(node.label))
```

Reply:
Thanks @dave-connors-3 - totally bizarre I'm sure I tried this one line fix the first time around and it didn't work, but running it again now it looks fine! 

Thank you for the advice on the integration test - sorry I'm not too familiar with them so wasn't sure how best to demonstrate it works, will add a new commit now :)